### DISCUSSION TITLE
Turning off Gpio monitor backlight while sleeping

### DISCUSSION THREAD

**Discussion by nikiwiener** [UnchartedBull/OctoDash#2517] [Ideas]
Hi!

Would it be possible to turn off the backlight of a monitor connected via the gpio pins while Octodash is sleeping? Currently the screen just turns black, but I image it cloud save some energy and prolong the screens life if turned off completely

**Comment by UnchartedBull** [comment]
Hi, sure it's possible. It will require some custom code though. I'm not really keen on merging this into main, since that might lock people out that aren't using a raspberry pi. Feel free to create your own fork and do the changes there though! If you need help let me know.
